sklearn/multioutput.py

```diff
@@ -301,15 +310,27 @@ class MultiOutputClassifier(ClassifierMixin, _MultiOutputEstimator):
         :term:`predict_proba`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation.
-        It does each target variable in y in parallel.
+        Pool size for multiprocessing / multithreading, passed to
+        ``joblib.Parallel()`` for use in :meth:`fit` and :meth:`predict`.
```

Comment:
I think this is kind of an implementation detail, we don't need to mention it. And we don't seem to in other places, e.g.: https://github.com/scikit-learn/scikit-learn/pull/15613/files

Reply:
I see your point, will modify. Thanks for comment!